dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/BasicDeploymentForm.tsx

```diff
@@ -141,7 +141,7 @@ class BasicDeploymentForm extends React.Component<IBasicDeploymentFormProps> {
         if (param.render === "textArea") {
           paramComponent = (
             <TextParam
-              label={param.title || name}
+              label={param.title || param.path}
```

Comment:
Sheesh - where was `name` even defined here?

Reply:
this is tricky, `name` is a global value (that resolves to `nodejs`, weird). It's difficult to notice because the linter didn't complain, now it seem it does :)